release.sh

```diff
@@ -1,29 +1,29 @@
-#!/bin/sh -e
+#!/bin/bash -e
 
 OTP_RAW_VER=$(erl -eval 'erlang:display(erlang:system_info(otp_release)), halt().'  -noshell)
-OTP_VER=$(echo ${OTP_RAW_VER} | sed -r 's/\"([0-9]+)\"/\1/g' | sed 's/[^0-9]*//g')
+OTP_VER=$(echo "${OTP_RAW_VER}" | sed -r 's/\"([0-9]+)\"/\1/g' | sed 's/[^0-9]*//g')
 
 yes_or_exit() {
     while true; do
         read -r -p "$1 " yn
         case $yn in
             [Yy][Ee][sS]) true; break ;;
-            [Nn][Oo]) echo "Ok, bye bye"; exit ;;
-            * ) echo "Please answer yes or no." ;;
+            [Nn][Oo]) echo "Ok, bye bye!"; exit ;;
+            * ) echo "Please answer 'yes' or 'no'." ;;
         esac
     done
 }
 
-if [ "${OTP_VER}" != "24" ]; then
-     echo "You are using Erlang/OTP ${OTP_VER_STR}"
-     echo "However, ex_doc must be built using OTP == 24 for compatability. Giving up..."
+if [ $((OTP_VER)) -lt 24 ]; then
```

Comment:
This doesn't play well with my bash shell, sadly.

Reply:
Hm... what version of `bash` are you using? The change I propose is, as far as I know, standard: https://www.gnu.org/software/bash/manual/html_node/Arithmetic-Expansion.html

Reply:
GNU bash, version 3.2.57(1)-release (arm64-apple-darwin22)

In retrospec, it wasn't the syntax, it was the `-lt 24` bit (i.e., I was testing it by running 26, but you `-lt 24`, so it didn't blow up as it should, I simply glazed over that detail at the time.